test/SerilogWeb.Classic.Tests/WebRequestLoggingHandlerTests.cs

```diff
@@ -363,6 +363,32 @@ public void PasswordBlackListCanBeCustomized()
             Assert.Equal(expectedLoggedData.ToString(), formDataProperty.ToString());
         }
 
+        [Fact]
```

Comment:
On the line just after the `[Fact]`, could you add `[Trait("Bugfix", "#62")]` ? 

This will help us to keep the correlation between the GitHub issue #62 and the test :)

Reply:
Will do.